### PR TITLE
Feature/#7 add edit view

### PR DESCRIPTION
**Pull request by jimitpatel9**

### CODE REVIEW

**Comment by jshor** [comment]
LGTM

File: client/vivace/details/controller.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+
+module.exports = function(dataStoreService) {
+  this.user = {
```
Comment (jackofseattle) [inline client/vivace/details/controller.js]: You don't need this.

File: test/client/vivace/add-edit/controller_test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by jimit on 12/15/15.
+ */
+/**
+ * Created by jimit on 12/15/15.
```
Comment (jackofseattle) [inline test/client/vivace/add-edit/controller_test.js]: Double header comments  - don't need them anyways

File: client/vivace/details/controller.js (hunk 1 above)
Comment (jimitpatel9) [inline client/vivace/details/controller.js]: you mean the controller or the `this` keyword??

File: test/client/vivace/details/controller_test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by jimit on 12/15/15.
+ */
```
Comment (jackofseattle) [inline test/client/vivace/details/controller_test.js]: Don't include these header comments.

Comment (jimitpatel9) [inline test/client/vivace/details/controller_test.js]: removed comments

**Comment by jackofseattle** [comment]
You have merge conflicts here now

**Comment by jimitpatel9** [comment]
Merge conflict solved

**Pull request merged**